packages/trader/src/AppV2/Components/TradeParameters/AllowEquals/allow-equals.tsx

```diff
@@ -0,0 +1,104 @@
+    const { contract_types_list, contract_start_type, duration_unit, expiry_type, is_equal, onChange } =
+        useTraderStore();
+
+    const [is_open, setIsOpen] = React.useState(false);
+    const [is_allow_equal_enabled, setIsAllowEqualEnabled] = React.useState(!!is_equal);
+
+    const has_callputequal_duration = hasDurationForCallPutEqual(
+        contract_types_list,
+        duration_unit,
+        contract_start_type
+    );
+    const has_callputequal = hasCallPutEqual(contract_types_list);
+    const has_allow_equals = (has_callputequal_duration || expiry_type === 'endtime') && has_callputequal;
+
+    const onSaveButtonClick = () => {
+        if (!!is_equal !== is_allow_equal_enabled)
+            onChange({ target: { name: 'is_equal', value: Number(is_allow_equal_enabled) } });
+    };
+    const onActionSheetClose = () => {
+        setIsOpen(false);
+        setIsAllowEqualEnabled(!!is_equal);
+    };
+
+    const action_sheet_content = [
+        {
+            id: 1,
+            component: (
+                <React.Fragment>
+                    <ActionSheet.Content className='allow-equals__wrapper'>
+                        <div className='allow-equals__content'>
+                            <Text>
+                                <Localize i18n_default_text='Allow equals' />
+                            </Text>
+                            <ToggleSwitch
+                                checked={is_allow_equal_enabled}
+                                onChange={(is_enabled: boolean) => setIsAllowEqualEnabled(is_enabled)}
+                            />
+                        </div>
+                    </ActionSheet.Content>
+                    <ActionSheet.Footer
+                        alignment='vertical'
+                        primaryAction={{
+                            content: <Localize i18n_default_text='Save' />,
+                            onAction: onSaveButtonClick,
+                        }}
+                    />
+                </React.Fragment>
+            ),
+        },
+        {
+            id: 2,
+            component: (
+                <ActionSheet.Content className='allow-equals__wrapper--definition'>
+                    <div className='allow-equals__content'>
+                        <Text>
+                            <Localize i18n_default_text='Win payout if exit spot is also equal to entry spot.' />
+                        </Text>
+                    </div>
+                </ActionSheet.Content>
+            ),
+        },
+    ];
+
+    React.useEffect(() => {
+        setIsAllowEqualEnabled(!!is_equal);
+    }, [is_equal]);
+
+    if (!has_allow_equals) return null;
+
+    return (
+        <React.Fragment>
+            <TextField
+                variant='fill'
+                readOnly
+                label={<Localize i18n_default_text='Allow equals' />}
+                value={is_equal ? localize('Enabled') : '-'}
```

Comment:
will it be <Localize/>?

Reply:
According to Quill, `value` can be `string | number | undefined`. 
<img width="850" alt="Screenshot 2024-07-23 at 8 41 11 AM" src="https://github.com/user-attachments/assets/4965bcc6-bcd0-46d5-8e40-6eec3e430cf7">